subprojects/core/build.gradle.kts

```diff
@@ -32,6 +32,8 @@ tasks.classpathManifest {
 }
 
 dependencies {
+    api(project(":file-temp"))
```

Comment:
This should be `implementation()` like all the other dependencies. We don't use `api()` propagation currently as we should. @jjohannes probably has some ideas about getting there, but until then we should keep the current pattern.

Reply:
Sounds good